### ISSUE TITLE
tick stick carpentry trick for copying a shape

### ISSUE THREAD

**Issue by stephanstan** [stephanstan/stephanstan.github.io#27]
## TL;DR
Little demonstrates how to use a little know tool named a "tick stick or ticking stick".  A tick stick is used to quickly and preciously copy and make a pattern of any odd or complicated shape where using a ruler would be challenging.  The ticking stick was made popular by boat-builders and is a way of measuring accurately which has been around for centuries.   
If you like our vids, treat Leah to a cup of coffee!
http://www.Ko-fi.com/seejanedrill

### Article Link
[tick stick video](https://www.youtube.com/watch?v=Cd2LY857oTY)
## Key Takeaways
* 
* 

## Useful Code Snippets
```python
#Add your codes here


```

## Useful Tools
* 
*
